test/controllers/adminorders_controller_test.rb

```diff
@@ -0,0 +1,7 @@
+require "test_helper"
+
+class AdminordersControllerTest < ActionDispatch::IntegrationTest
+  # test "the truth" do
+  #   assert true
+  # end
+end
```

Comment:
LGTM, just this suggestion

```suggestion
class AdminordersControllerTest < ActionDispatch::IntegrationTest
end
```